components/function-controller/internal/controllers/serverless/fsm.go

```diff
@@ -51,7 +51,8 @@ type reconciler struct {
 }
 
 const (
-	continuousGitCheckoutLabel = "serverless.kyma-project.io/continuousGitCheckout"
+	continuousGitCheckoutLabel     = "serverless.kyma-project.io/continuousGitCheckout"
```

Comment:
We should probably change the `Label` suffix

Reply:
Yup, thanks. This change is committed in [this PR](https://github.com/kyma-project/kyma/pull/14671/files#diff-76e76f20345ca2b85f060c5e41f385358d07742aa541c9f6c70967b3063e59cbL54)